operator/pkg/apis/operator/v1alpha1/type.go

```diff
@@ -557,12 +557,11 @@ type HostCluster struct {
 type ConditionType string
 
 const (
-	// Unknown represent a condition type the karmada not be reconciled by operator
-	// or unpredictable condition.
-	Unknown ConditionType = "Unknown"
-
 	// Ready represent a condition type the all installtion process to karmada have compaleted.
 	Ready ConditionType = "Ready"
+
+	// Shutdown represent a condition type the uninstall process to karmada have complaleted.
+	Shutdown ConditionType = "Shutdown"
```

Comment:
Seems that `Ready` means in the progressing of installing, `Shutdown` means in the progressing of uninstalling.

I don't think these words are clear enough.

Reply:
Yes, I've been thinking about it for a long time.  I have some confusion here：

*   Do both of installation and uninstallation use the same condition type `Ready`？
*   If use a different ccondition type, these words are not clear enough.

Do you have any good suggestions?

Reply:
When the condition would be `Shutdown`? Does deleting the `Karmada` means shut down?

Reply:
When removing Karmada CR and triggers the process of uninstalling karmada, which takes about 5 seconds. In this short time, I want to use condition tyepe `shutdown` to represent the current state of karmada.  or is there a better condition type define?

Reply:
> Controllers should apply their conditions to a resource the first time they visit the resource, even if the status is Unknown. This allows other components in the system to know that the condition exists and that the controller is making progress on reconciling that resource.

Sorry for late reply, ok, I got that means and I'm agress with you.

I think we should only need one condition type (`Ready` or other) to define the whole karmada lifecycle. for different operation  `init`, `upgrade` or `deint` denoted by different reason. for example:

Deletion operation condition:

```yaml
type: Ready
status: False
reason: Deleting
message: "the karmada is being deleted"
```

Reply:
Then, we don't need the `ShutDown` condition, right? Please remove it.

Reply:
yes, I think so.